generated/nidigital/session.py

```diff
@@ -2770,6 +2770,85 @@ def export_signal(self, signal, signal_identifier, output_terminal):
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return
 
+    @ivi_synchronized
+    def _fetch_capture_waveform(self, site_list, waveform_name, samples_to_read, timeout):
+        # This is slightly modified codegen from the function
```

Comment:
Why wasn't this code-generated?

Reply:
Because the code generator doesn't support an 'ivi-dance' where there are two values returned and then multiplied together.